File: 2-Anno/Linguaggi/Java/Pratica/Simulazioni/N40/Veicolo.java
package N40;

public class Veicolo {
	
	private int codice;
	private String tipo;
	private String targa;
	private String modello;
	private String marca;
	private double costoGiornaliero;


	public Veicolo(int codice, String tipo, String targa, String modello, String marca, double costoGiornaliero) {
		this.codice = codice;
		this.tipo = tipo;
		this.targa = targa;
		this.modello = modello;
		this.marca = marca;
		this.costoGiornaliero = costoGiornaliero;
	}

	public String getTarga() {
		return targa;
	}

	public double getCosto() {
		return costoGiornaliero;
	}

	public String toString() {
		if (tipo.equals("auto")) {
			return "automobile" + "\t" + targa + "\t" + codice + "\t" + modello + "\t" + marca + "\t" + costoGiornaliero;
		}

		return "commerciale" + "\t" + targa + "\t" + codice + "\t" + modello + "\t" + marca + "\t" + costoGiornaliero;
	}
}

File: 2-Anno/Linguaggi/Python/Pratica/Simulazioni/N40/Veicolo.py
class Veicolo:
	
	def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero) -> None:
		self._codice = codice
		self._tipo = tipo
		self._targa = targa
		self._modello = modello
		self._marca = marca
		self._costoGiornaliero = costoGiornaliero

	def getTarga(self) -> str:
		return self._targa

	def getCosto(self) -> float:
		return self._costoGiornaliero

	def __str__(self) -> str:
		
		if self._tipo == "auto":
			return "automobile" + "\t" + str(self._targa) + "\t" + str(self._codice) + "\t" + str(self._modello) + "\t" + str(self._marca) + "\t" + str(self._costoGiornaliero)
		
		return "commerciale" + "\t" + str(self._targa) + "\t" + str(self._codice) + "\t" + str(self._modello) + "\t" + str(self._marca) + "\t" + str(self._costoGiornaliero)